src/templates/loading_page.hdbs

```diff
@@ -9,7 +9,7 @@
           <div class="u-fontSize-big u-lineHeight-5x u-flexGrow-1 u-color-primary">&nbsp;</div>
 
           <div class="u-textAlign-right u-display-flex u-alignItems-center">
-
+            <div style="width: 1px; height: 41px;"></div>
```

Comment:
I am very fond of inline styles but I see we use them elsewhere too. If @codegaze approves of this, I am fine with it.